requirements.txt

```diff
@@ -3,7 +3,7 @@ aiohttp
 altair>=4.2.0
 fastapi
 ffmpy
-gradio_client>=0.2.4
```

Comment:
Thanks for the fix @WilliamRoyNelson ! 

This looks good to me but I don't think we actually don't have to bump the version in requirements.txt. gradio will still work with version 0.2.4.